### ISSUE TITLE
Cpu autodetection

### ISSUE THREAD

**Issue by jbenden** [jbenden/aircrack-ng-migration#1667]
**Reported by ilvalle on 2 Feb 2016 20:19 UTC**

On a ubuntu with kernel 4.2.0-27-generic, I got the following
```
$$ aircrack-ng -u
Vendor          = Intel
Model           = Intel(R) Core(TM) i5-2450M CPU @ 2.50GHz
Features        = MMX,SSE,SSE2,SSE3,SSSE3,SSE4.1,SSE4.2,AES-NI,AVX
Hyper-Threading = No
CPU cores       = 4
SIMD size       = 4 (128 bit)
```
However according to http://ark.intel.com/products/53452/Intel-Core-i5-2450M-Processor-3M-Cache-up-to-3_10-GHz Hyper-Threading is supported

Migrated-From: https://trac.aircrack-ng.org/ticket/1667

**Comment by jbenden** [comment]
**Comment by misterx on 2 Feb 2016 20:19 UTC**

Have you checked if it isn't disabled in the BIOS?

**Comment by jbenden** [comment]
**Comment by misterx on 2 Feb 2016 20:19 UTC**

Should be fixed by aircrack-ng/aircrack-ng@bb555c9f9f4d0d534b7ca0231e0abf5b6176b2ea.
